mdsf/src/tools/beautysh.rs

```diff
@@ -33,20 +33,20 @@ pub fn run(file_path: &std::path::Path) -> Result<(bool, Option<String>), MdsfEr
 #[cfg(test)]
 mod test_beautysh {
     #[test_with::executable(beautysh)]
-    fn test_beautysh_shell_cb36b80a8b253f58() {
-        let input = r#"#!/bin/shell
+    fn test_beautysh_bash_a6831a7ad31bd0a6() {
```

Comment:
Are this file and others auto generated?

I mean partially or fully ?

I think they are generated because of lines like this.

I'm asking because I wounded expect auto generated files to be flagged with .gitattributes

This way GitHub will mark them as auto generated and will exclude them from diff

Also a generated file often has an exit like DO NOT EDIT; auto generated files

Something IDEs detect preventing anyone to edit these files

Reply:
Yes, all the files in `mdsf/src/tools` are generated based on the `plugin.json` files found in `tools/`. 

I have not flagged the with gitattributes since I am interested in tracking differences between commits. 

Adding a comment to the files makes a lot of sense!

Reply:
I have added a comment to all the generated files! 😄